marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/flow/testTargetCollectionAdditivity/test-data/entities/Customer.entity.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
can you remove all the entities/flow/step and try to reuse use the new createSimpleMappingProject helper in data-hub-test-helper.sjs ?

Reply:
Definitely - should be fine to just use a mapping step as part of this. You can have a document under test-data/content, which means it'll have a collection of "raw-content" on it. And you can verify that it's either preserved or not preserved.